bin/specextract

```diff
@@ -1326,8 +1374,8 @@ def get_filename(full_filename):
 
 
 
-    if not filename or str(filename) in ["NULL", "NONE", "None",""," "]:
-        return NULL
+    if not filename or str(filename) in [None, "NULL", "NONE", "None",""," "]:
+        return None
 
     else:
```

Comment:
pylint (or pylakes or ....) suggests it's bad form to have logic like

```
if some_conditon:
    ... code
    return value1
elsr:
    ... other code
    return value2
```

since the claim is it's better to say

```
if some conditon:
    ... code
    return value1

... other code
return value2
```